wom/src/main/scala/wom/callable/CommandTaskDefinition.scala

```diff
@@ -174,3 +173,44 @@ final case class ExecutableTaskDefinition private (callableTaskDefinition: Calla
   override private [wom]  def customizedOutputEvaluation = callableTaskDefinition.customizedOutputEvaluation
   override def toExecutable = this.validNel
 }
+
+sealed trait ExpressionTaskDefinition extends TaskDefinition {
+  def evaluate: (Map[String, WomValue], IoFunctionSet, List[OutputPort]) => Checked[Map[OutputPort, WomValue]]
+}
+
+
+/**
+  * An expression task definition only.
+  * Can be called but cannot be used in an Executable as a standalone execution.
```

Comment:
> cannot be used in an Executable as a standalone execution

(Tol as I don't understand Expression Tools yet) do we enforce this?

Reply:
I actually copied this from the `CommandLineTool` version.
It's a `WOM` concept.
`Executable`s can be executed as "standalone" objects.
`Callable`s can be called in a workflow step.
`ExecutableCallable`s are both.
AFAIK It's enforced in the sense that Cromwell gives WOM basically 2 `String`a (workflow and inputs) and expects an `Executable` to come back.